hotness/buildsys.py

```diff
@@ -60,7 +62,14 @@ def __init__(self, consumer, config):
 
     def session_maker(self):
         koji_session = koji.ClientSession(self.server, {'timeout': 3600})
```

Comment:
You need a krb_sessionopts probably, or hardcode {'timeout': 3600} to be {'timeout': 3600, 'krb_rdns': False}.
krb_rdns set to False is a requirement in the Fedora Koji krb auth.